awesome_sso/service/user/sync_user.py

```diff
@@ -1,12 +1,10 @@
-from typing import Type
-
 import requests
 
 from awesome_sso.service.settings import Settings
 from awesome_sso.service.user.schema import AwesomeUserType
 
 
-async def sync_user(user: Type[AwesomeUserType]):
+async def sync_user(user: AwesomeUserType):
```

Comment:
I don't quite understand this part of the change? If we intend sync_user to be a generic function shouldn't we have a Type wrapped around AwesomeUserType?